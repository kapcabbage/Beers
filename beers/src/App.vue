<template>
	<v-app dark id="app">
    
		<v-content>
       <router-view></router-view>
		</v-content>
		<v-footer class="xs12 d-block pa-2 text-xs-center">
			Beers &copy; {{ new Date().getFullYear() }}
		</v-footer>
	</v-app>
</template>

<script>
import { mapGetters } from 'vuex';

export default {
  name: "App",
  components: {
     
  },
  data() {
    return{

    }
  },
  computed: {
    
  },

};
</script>

<style lang="scss">
  @import './styles/main.scss';
</style>