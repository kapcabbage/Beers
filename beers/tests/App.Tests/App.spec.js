import {shallowMount,createLocalVue} from '@vue/test-utils';
import Vuex from 'vuex';
import Vuetify from 'vuetify';
import Vue from 'vue';
import App from '../../src/App.vue';
import VueRouter from 'vue-router'

const localVue = createLocalVue()

localVue.use(Vuex)
localVue.use(VueRouter)
Vue.use(Vuetify);
describe('App.vue', () => {
    let wrapper;
    beforeEach(() => {
        wrapper = shallowMount(App,{
            localVue
        });
    });
    it('Does *App* page render correctly', () => {
        //assert
        expect(wrapper.vm.$el).toMatchSnapshot();
    });
})