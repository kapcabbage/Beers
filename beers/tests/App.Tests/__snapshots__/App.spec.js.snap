// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App.vue Does *App* page render correctly 1`] = `
<v-app-stub
  dark="true"
  id="app"
>
  <navigationdrawer-stub
    computeddrawer="true"
  />
   
  <v-content-stub
    tag="main"
    transition="slide-x-transition"
  >
    <router-view-stub
      name="default"
    />
  </v-content-stub>
   
  <v-footer-stub
    class="xs12 d-block pa-2 text-xs-center"
    height="32"
  >
    
			Arvato Â© 2019
		
  </v-footer-stub>
</v-app-stub>
`;
